Frontend/CO.CDP.OrganisationApp/Pages/Consortium/ConsortiumEmail.cshtml.cs

```diff
@@ -0,0 +1,104 @@
+using CO.CDP.Localization;
+using CO.CDP.Mvc.Validation;
+using CO.CDP.Organisation.WebApiClient;
+using CO.CDP.OrganisationApp.Constants;
+using CO.CDP.OrganisationApp.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.FeatureManagement.Mvc;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using OrganisationWebApiClient = CO.CDP.Organisation.WebApiClient;
+
+namespace CO.CDP.OrganisationApp.Pages.Consortium;
+
+[FeatureGate(FeatureFlags.Consortium)]
+[ValidateConsortiumStep]
+public class ConsortiumEmailModel(
+    ISession session,
+    IOrganisationClient organisationClient) : ConsortiumStepModel(session)
+{
+    public override string CurrentPage => ConsortiumEmailPage;
+
+    [BindProperty]
+    [DisplayName(nameof(StaticTextResource.Consortium_ConsortiumEmail_Heading))]
+    [Required(ErrorMessageResourceName = nameof(StaticTextResource.Consortium_ConsortiumEmail_Required_ErrorMessage), ErrorMessageResourceType = typeof(StaticTextResource))]
+    [ValidEmailAddress(ErrorMessageResourceName = nameof(StaticTextResource.Global_Email_Invalid_ErrorMessage), ErrorMessageResourceType = typeof(StaticTextResource))]
```

Comment:
Check with @maciej-goaco but I think `ValidEmailAddress` is meant to work in concert with the `SanitisedStringModelBinder`

```suggestion
    [ModelBinder<SanitisedStringModelBinder>]
    [ValidEmailAddress(ErrorMessageResourceName = nameof(StaticTextResource.Global_Email_Invalid_ErrorMessage), ErrorMessageResourceType = typeof(StaticTextResource))]
```